group_doc

```diff
@@ -0,0 +1,35 @@
+#-----------------------------------------------------Part 2; Exercise #1
+#submitted by: Laura
+#updated by:
+#verified by: *needs review*
+
+SELECT product_code, product_name, list_price, discount_percent
+  FROM products;
+  
+SELECT product_code, product_name, list_price, discount_percent
+  FROM products
+ ORDER BY list_price;  
+ 
+#------------------------------------------------------------------------
+
+#-----------------------------------------------------Part 2; Exercise #2
+#submitted by: Laura
+#updated by:
+#verified by: *needs review*
+
+SELECT CONCAT(last_name, ', ', first_name) AS full_name
+  FROM customers
+ WHERE LEFT(last_name, 1) BETWEEN 'M' and 'Z'
+ ORDER BY last_name;
```

Comment:
Very interesting approach! My solution returns the same information, but is formatted slightly differently:

SELECT concat_ws(", ", last_name, first_name) AS full_name
FROM customers
WHERE last_name >= "M%"
ORDER BY last_name ASC;

Reply:
Your approach is good too. I figured we would have a lot of this because there are so many ways to accomplish the same thing...which is good because we need updates/edits in order to get credit for enough work per person on this project. The question I have on using 'M%' is that there are plenty of ASCII characters after Z and the problem states 'from M to Z'. Also, it now occurs to me that neither of our solutions are currently picking up a name with a typo where they forgot to capitalize the first character. do we need to add whatever the MySQL version of 'UPPER' is before the comparison?